src/shelf_usage/zone.cc

```diff
@@ -89,11 +89,6 @@ inline uint64_t next_power_of_two(uint64_t n)
 
 #define BYTE 8UL
 #define ATOMIC_SIZE 64UL
-#define KB 1024UL
-#define MB (KB * KB)
-#define GB (MB * KB)
-//#define TB (GB * KB)
-#define MAX_ZONE_SIZE (1024 * GB)
 #define MIN_OBJ_SIZE 64
```

Comment:
I think it would be good to move the rest of commonly used #defines in the common.h header file, unless they are specific to this module.